llvm/CMakeLists.txt

```diff
@@ -293,7 +293,9 @@ set(LLVM_ALL_TARGETS
   AMDGPU
   ARM
   AVR
-  BPF
+  # These tests are currently broken on upstream. Disable them so we can still
```

Comment:
It could be that they are broken for our particular setup, so I wonder if something like this is better:

```
The BPF tests currently fail on our CI machine. This could be due to the kernel version that we use, but either way, the tests also fail on vanilla LLVM, so the failures are not due to our changes.
```

Reply:
Try 9463808.